api/HTMLMediaElement.json

```diff
@@ -1436,7 +1436,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.6"
+              "version_added": "3.5"
```

Comment:
Makes sense, 3.5 is when the rest of the initial `HTMLMediaElement` was shipped.